widgets/ConversationView.php

```diff
@@ -33,6 +34,19 @@ class ConversationView extends JsWidget
      */
     public $messageId;
 
+    public function init()
+    {
+        parent::init();
+
+        $this->view->registerJsConfig([
+            'mail.ConversationView' => [
+                'text' => [
+                    'warn.onBeforeUnload' => Yii::t('base', 'Unsaved changes will be lost. Do you want to proceed?'),
```

Comment:
@gevorgmansuryan Why do we need to specify an own message here? 

It would be nice if the core message could simply be used here. Without being defined twice.

(If we change the core message later, we would also have to change it in the mail module).